screenpy/actions/__init__.py

```diff
@@ -27,6 +28,7 @@
 Sleeps = Pauses = Pause
 TakeNote = MakeNote
 TakesNote = MakesNote = MakeNote
+Either = SeeIfTheyCan = Attempt = AttemptTo = GoFor = Try = TryTo
```

Comment:
I'm not sure i like the `SeeIfTheyCan` alias, not just because it's pretty long—`See` makes it sound like an assertion to me. We've got 3 `See` Actions, i don't want folks to accidentally reach for this one. I think you might want to pare down the number of aliases you go for here, 'cause we need to support several tenses now.

However, `Either`, i think i like that one. That fits alongside `Eventually` as one of the odd-duck adverb-style Actions and works with all of our tenses:
```python
the_actor(Either(MakesNote.of_the(...)).or_(Sleeps())
the_actor.will(Either(MakeNote.of_the(...)).or_(Sleep())
the_actor.attempts_to(Either(MakeNote.of_the(...)).or_(Sleep())
```

What are your thoughts on just using `Either` as the name for this new Action?

Reply:
I think what put me off of a lot of these aliases is thinking of writing
```python
the_actor.attempts_to(AttemptTo(...))
the_actor.attempts_to(TryTo(...))
```

Reply:
> I think what put me off of a lot of these aliases is thinking of writing

I was thinking about the option to write the following:
`actor.will(AttemptTo())` 

> What are your thoughts on just using Either as the name for this new Action?

I'm not opposed to that.  TryTo felt like the right base name since folks generally understand what `try` does in python.  In reality this is basically a glorified try/except.

> I'm not sure i like the SeeIfTheyCan alias...

I had the same thought.  I threw it in there to test the waters.  I keep feeling like there might be another way to state things in a similar manner without using 'See'.

Reply:
I am finally coming around to changing `TryTo` to `Either` but now I'm faced with a new challenge.  I wish the describe text would change with the aliases.

Reply:
What about this?
```python
class TryTo:
    ...
    def describe(self) -> str:
        ...

class Either(TryTo):
    def describe(self) -> str:
        ...

class Attempt(TryTo):
    def describe(self) -> str:
        ...

class GoFor(TryTo):
    def describe(self) -> str:
        ...
```

this would also let us set the contextual aliases for `or_`, like only `GoFor` would have `failing_that` or something.

Reply:
For the moment, I'm not going to address the alias issue.  #93 can make those adjustments as we figure out how we handle that.